modules/user_accounts/test/user_accountsTest.php

```diff
@@ -104,50 +103,114 @@ function testUserAccountsMyPreferencesDoespageLoad()
+                 input.dispatchEvent(event);
+                "
+            );
+                    $bodyText = $this->webDriver->executescript(
+                        "return document.querySelector".
+                          "('#userAccounts_dataTable > div.table-header".
+                          " > div > div > div:nth-child(1)').textContent"
+                    );
+                    // 4 means there are 4 records under this site.
+                    $this->assertContains($records, $bodyText);
+        }
+        //test clear filter
+        $btn = $this->_clearFilter;
+        $this->webDriver->executescript(
+            "document.querySelector('$btn').click();"
+        );
+        $inputText = $this->webDriver->executescript(
+            "return document.querySelector('$element').value"
+        );
+        $this->assertEquals("", $inputText);
+    }
+
+
     /**
      * Tests various user account edit operations.
      *
      * @return void
      */
     function testUserAccountEdits()
     {
-        $this->markTestSkipped("Skipping excessively slow test");
-        return;
         $this->_verifyUserModification(
             'user_accounts',
             'UnitTester',
             'First_name',
             'NewFirst'
         );
-        $this->_verifyUserModification(
-            'user_accounts',
-            'UnitTester',
-            'Last_name',
-            'NewLast'
-        );
-        $this->_verifyUserModification(
-            'user_accounts',
-            'UnitTester',
-            'Active',
-            'No'
-        );
-        $this->_verifyUserModification(
-            'user_accounts',
-            'UnitTester',
-            'Email',
-            'newemail@example.com'
-        );
-        $this->_verifyUserModification(
-            'user_accounts',
-            'UnitTester',
-            'Pending_approval',
-            'No'
-        );
+        //  these tests below needs more time to run
+        //  $this->markTestSkipped("Skipping excessively slow test");
+        //  return;
+        //  $this->_verifyUserModification(
+        //      'user_accounts',
+        //      'UnitTester',
+        //      'First_name',
+        //      'NewFirst'
+        //  );
+        //  $this->_verifyUserModification(
+        //      'user_accounts',
+        //      'UnitTester',
+        //      'Last_name',
+        //      'NewLast'
+        //  );
+        //  $this->_verifyUserModification(
+        //      'user_accounts',
+        //      'UnitTester',
+        //      'Active',
+        //      'No'
+        //  );
+        //  $this->_verifyUserModification(
+        //      'user_accounts',
+        //      'UnitTester',
+        //      'Email',
+        //      'newemail@example.com'
+        //  );
+        //  $this->_verifyUserModification(
+        //      'user_accounts',
+        //      'UnitTester',
+        //      'Pending_approval',
+        //      'No'
+        //  );
```

Comment:
Why are these tests commented out?

Reply:
because @kongtiaowang amended it that way